test/util/cluster/cluster.go

```diff
@@ -22,7 +22,7 @@ type EnabledNodes struct {
 var (
 	supportedPFDrivers = []string{"mlx5_core", "i40e", "ixgbe"}
 	supportedVFDrivers = []string{"iavf", "vfio-pci", "mlx5_core"}
-	supportedDevices   = []string{"1583", "158b", "10fb", "1015", "1017"}
+	supportedDevices   = []string{"1583", "158b", "10fb", "1015", "1017", "1572"}
```

Comment:
Hmm, i dont see `supportedDevices` in master branch 

https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/348e699badfaf15d30fe2c68bfd1f07a4b0e0c10/test/util/cluster/cluster.go#L29